File: src/python/py-accepted/158A.py
#! python

# Problem #    : 158A
# Created on   : 2019-01-14 20:45:14

def Main():
    n, k = map(int, input().split(' '))
    arr = list(map(int, input().split(' ')))
    cnt = 0
    for i in arr:
        if i >= arr[k - 1] and i > 0:
            cnt += 1
    
    print(cnt)
        

if __name__ == '__main__':
    Main()

File: js-accepted/158A.js
// Problem #    : 158A
// Created on   : 2019-01-14 20:09:04

var arr = readline().split(" ");
var n = arr[0];
var k = arr[1];

var l = readline().split(" ");
var kth = Number(l[k - 1]);
var cnt = 0;

for (var i = 0; i < l.length; i++) {
    var x = l[i];
    if (x >= kth && x > 0) {
        cnt++;
    }
}
print(cnt);